ruleset/testing/tests/exim.ini

```diff
@@ -1,42 +1,42 @@
-[auth failure]
+;  Copyright (C) 2015-2021, Wazuh Inc.
+;
+;  Tests for products:
+;    Exim 4
+;
+
+[exim auth failure]
 log 1 pass = 2017-01-23 03:44:14 dovecot_login authenticator failed for (hydra) [10.101.1.18]:35686: 535 Incorrect authentication data (set_id=user)
 log 2 pass = 2017-01-24 05:22:29 dovecot_plain authenticator failed for (test) [::1]:39454: 535 Incorrect authentication data (set_id=test)
-
-rule = 87506
+rule = 87502
 alert = 5
 decoder = windows-date-format
 
 [exim connection]
 log 1 pass = 2017-01-24 03:09:46 SMTP connection from [10.101.1.10]:55010 (TCP/IP connection count = 1)
-
-rule = 87508
+rule = 87504
 alert = 0
 decoder = windows-date-format
 
 [exim connection lost]
 log 1 pass = 2017-01-24 02:53:13 SMTP connection from (hydra) [10.101.1.10]:53682 lost
-
-rule = 87509
+rule = 87505
 alert = 1
 decoder = windows-date-format
 
 [exim syntax/protocol error]
 log 1 pass = 2017-01-24 05:36:23 SMTP call from (000000) [::1]:39480 dropped: too many syntax or protocol errors (last command was "123")
-
-rule = 87510
+rule = 87506
 alert = 5
 decoder = windows-date-format
 
 [exim protocol synchronization error]
 log 1 pass = 2019-10-20 11:14:38 SMTP protocol synchronization error (input sent without waiting for greeting): rejected connection from H=[134.234.45.34] input="GET / HTTP/1.1\r\nHost: 24.255.212.213:98\r\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:47.0) Gecko/20100101 Firefox/47.0\r\nAccept: */*\r\n"
-
-rule = 87511
+rule = 87507
 alert = 6
 decoder = windows-date-format
 
 [exim Unrouteable address]
 log 1 pass = 2019-10-20 09:56:39 H=123-123-12-123.example.example.net [123.123.12.123] F=<example@example.com> rejected RCPT <example@exampley.com>: Unrouteable address
-
-rule = 87512
+rule = 87508
 alert = 6
 decoder = windows-date-format
```

Comment:
Are the rule's ID changes on purpose?

Reply:
They were reordered [here](ac926924bfb0e520e889c248ef30644e54261df4) to avoid jumps in the rule IDs

Reply:
Keep in mind that rule id changes of existing rules must be "triple checked"